ui/src/adapters/api/credentials.ts

```diff
@@ -30,19 +31,17 @@ import {
 import { API_VERSION, QUERY_SEARCH_PARAM, STATUS_SEARCH_PARAM } from "src/utils/constants";
 import { List, Resource } from "src/utils/types";
 
-type ProofTypeInput = "BJJSignature2021" | "SparseMerkleTreeProof";
-
-const proofTypeParser = getStrictParser<ProofTypeInput[], ProofType[]>()(
+const proofTypeParser = getStrictParser<CredentialProofType[], ProofType[]>()(
   z
-    .array(z.union([z.literal("BJJSignature2021"), z.literal("SparseMerkleTreeProof")]))
+    .array(z.nativeEnum(CredentialProofType))
     .min(1)
     .transform((values) =>
       values.map((value) => {
         switch (value) {
```

Comment:
we should always display the correct type of Proof. They should not be replaced with SIG/MTP. 
It should be the proper full type `BJJSignature2021`, `Iden3SparseMerkleTreeProof` or other type of proof form the credential `proof` section

Reply:
Let's remove the type `ProofType` and use the enum `CredentialProofType` from the SDK everywhere, as we do in the web wallet.

Reply:
I just checked and no, we don't even have the JS SDK as a dependency on the schema builder yet. Let's leave it as you did it for now, but this needs to be replaced by the SDK enum along with several more changes.

Reply:
We should replace these references too:

<img width="247" alt="image" src="https://github.com/user-attachments/assets/d8343f1f-e0e2-4dee-8c94-7e1c6ae6b905">

_________

<img width="620" alt="image" src="https://github.com/user-attachments/assets/9277b96c-2d22-4271-a8f0-8c98109f5d12">

Reply:
<img width="698" alt="image" src="https://github.com/user-attachments/assets/2cb0297a-15e0-4e77-8f6c-90e32051b8fd">
it's already renamed in credential link details, or do you mean that we need to remove (SIG) and (MTP) in form?

Reply:
Yes, everywhere. Consistency is crucial.